docs/how-to/code-generator-version-control.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Code Generator Version Control
+description: This feature gives you the flexibility to chose which version of the code generator you want to use.
+sidebar_label: Code Generator Version Control
+slug: /code-generator-version-control
+---
+
+The **Code Generator Version Control** feature gives Pro and Enterprise plan users the flexibility to choose which version of the code generator they want to use.
+
+Amplication's code generator periodically releases new versions with bug fixes, enhancements, and new features. 
+
+While it's recommended to use the latest version, some teams may prefer having more control over when to upgrade the generated code.
+
+![Amplication Code Generator Version Control](./assets/code-generator-version-control.png)
+
+## Benefits
+
+- **Avoid Breaking Changes**: Upgrade on your own schedule and don't be forced to use a version that might contain breaking changes right away.
+
+- **Test New Versions**: Try out a new version and see how it impacts your app before rolling it out to production.
+
+- **Lock Down Stable Versions**: Lock in a stable version and keep generating code without unexpected changes.
+
+## How It Works
+
+1. In your service's **Settings** page, navigate to the **Code Generator Version** tab.
+
+:::note
+Only Code Generator versions that have been released since you created your service can be used. You can't use code generator versions that were released earlier than your service's creation date.
+:::
```

Comment:
@mulygottlieb @abrl91 @overbit  is that correct?

Reply:
@dericksozo This note should be removed. I am not aware of such limitation (@abrl91 @overbit feel free to correct me if I'm missing anything)